src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java

```diff
@@ -28,32 +28,30 @@
  * @author Josue Rodriguez, Alejandro Sánchez Luizaga
  * @version 1.0
  */
-public class ConvertPPTtoImage implements IConvert{
+public class ConvertPptxToImage implements IConvert{
     /**
      * Implements convert(CriteriaConvert) from IConvert interface.
      * Conversion is performed via Apache poi utility.
      *
      * @param criteriaConvert holds source and destination file paths and output file extension.
      */
     public void convert(CriteriaConvert criteriaConvert) {
-        CriteriaPPTtoImage criterion = (CriteriaPPTtoImage) criteriaConvert;
+        CriteriaPptxToImage criteria = (CriteriaPptxToImage) criteriaConvert;
         try {
-            File file = new File(criterion.getSrcPath());
+            File file = new File(criteria.getSrcPath());
             XMLSlideShow ppt = new XMLSlideShow(new FileInputStream(file));
             Dimension pgsize = ppt.getPageSize();
             List<XSLFSlide> slide = ppt.getSlides();
             BufferedImage img = null;
             for (int i = 0; i < slide.size(); i++) {
```

Comment:
Please the variable change for one more explicit.

Reply:
variable?

Reply:
the variable "i" to the for I think you cant use for example index.

Reply:
fix complete.